### PR TITLE
fix(data): corrections to Bhai Gurdaas Ji Vaaran

### PR DESCRIPTION
**Pull request by bhajneet**
#Patch

### CODE REVIEW

File: data/Vaaran Bhai Gurdas Ji/41.json [hunk 1]
```diff
@@ -125,7 +125,7 @@
         "id": "MGZC",
         "source_page": 41,
         "source_line": 1,
-        "gurmukhi": "guru ismr mnweI kwlkw; KMfy kI vylw [",
+        "gurmukhi": "guru dws mnweI kwlkw; KMfy kI vylw [",
```
Comment (Sarabveer) [inline data/Vaaran Bhai Gurdas Ji/41.json]: Should be `ismir`

File: data/Vaaran Bhai Gurdas Ji/41.json [hunk 2]
```diff
@@ -79,7 +79,7 @@
         "id": "3GE9",
         "source_page": 41,
         "source_line": 1,
-        "gurmukhi": "hir scy qKq rcwieAw; siq sMgiq mylw [",
+        "gurmukhi": "hir scy qKq rcwieAw; siqsMgiq mylw [",
```
Comment (Sarabveer) [inline data/Vaaran Bhai Gurdas Ji/41.json]: `siq sMgiq`

Comment (bhajneet) [reply data/Vaaran Bhai Gurdas Ji/41.json]: https://github.com/ShabadOS/database/pull/1009/commits/84c7ea880c96d8691c1336515f0329110db635ba

File: data/Vaaran Bhai Gurdas Ji/41.json (hunk 1 above)
Comment (bhajneet) [reply data/Vaaran Bhai Gurdas Ji/41.json]: this was a mistake, i forgot to discard changes before committing the next line, will do properly now

Comment (bhajneet) [reply data/Vaaran Bhai Gurdas Ji/41.json]: https://github.com/ShabadOS/database/pull/1009/commits/29f0ffef498ed65b7c81fc3c3e378c057bf8c94c & https://github.com/ShabadOS/database/pull/1009/commits/2b7d3811543ce29ed1601dc3b46affcb6b889b61

File: data/Vaaran Bhai Gurdas Ji/41.json [hunk 3]
```diff
@@ -125,7 +125,7 @@
         "id": "MGZC",
         "source_page": 41,
         "source_line": 1,
-        "gurmukhi": "guru dws mnweI kwlkw; KMfy kI vylw [",
+        "gurmukhi": "guru ismr mnweI kwlkw; KMfy kI vylw [",
```
Comment (Sarabveer) [inline data/Vaaran Bhai Gurdas Ji/41.json]: `ismir` as per SGPC Proof Footnote

**Comment by Harjot1Singh** [comment]
Note: This PR should be rebased

**Pull request merged**